recipe/meta.yaml

```diff
@@ -1,32 +1,38 @@
 {% set name = "sentence-transformers" %}
-{% set version = "2.0.0" %}
+{% set version = "2.2.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 492eae8e9973f2b758d66d74325661e2816f0b6637365554a1eca9bcf60088ed
+  sha256: dbc60163b27de21076c9a30d24b5b7b6fa05141d68cf2553fa9a77bf79a29136
 
 build:
+  # skip s390x and ppc64le because torchvision does not support these
+  skip: True  # [linux and (s390x or ppc64le)]
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -vvv --no-deps --no-build-isolation
   noarch: python
```

Comment:
```suggestion
```

Reply:
This is why I'm not blocking this, but I see it this way: the recipe reflects upstream requirements (plus our own build standards). Which python versions we build for is set by our `conda_build_config.yaml` (i.e., our config files).

Reply:
oh ok, got it. So in this case you think it's appropriate to add `skip: True [py<36]`? Just making sure that's what you meant.

Reply:
Yes, that is what I meant.